relay_client/src/http.rs

```diff
@@ -122,14 +123,15 @@ impl Client {
         &self,
         pairing_topic: Topic,
         session_topic: Topic,
-        pairing_response: impl Into<Arc<str>>,
+        session_proposal_response: impl Into<Arc<str>>,
         session_settlement_request: impl Into<Arc<str>>,
     ) -> Response<rpc::ApproveSession> {
         self.request(rpc::ApproveSession {
             pairing_topic,
             session_topic,
-            pairing_response: pairing_response.into(),
```

Comment:
was the name wrong?

Reply:
Not that it was 'wrong', but @llbartekll suggested a better alternative, more consistent with other fields there.